Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11.4-slim
+LABEL mantainer="volodymyr_dolgyi@ukr.net"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```

Comment:
Ensure that the `app/main.py` file exists and is correctly implemented as expected by the task requirements. The file should contain the script for getting and printing weather information for Paris using the Weather API.